docs/core/extensions/cloud-service.md

```diff
@@ -187,7 +187,7 @@ Replace placeholders with your own appropriate values:
 The preceding command:
 
 - Creates an Azure Container Registry, given a registry name, in the specified resource group.
-- Enabled an Admin user—this is required for Azure Container Instances.
+- Enabled an Admin user—this is required for Azure container instances.
```

Comment:
I disagree with this one, it should remain as a proper name, see: https://learn.microsoft.com/azure/container-instances/container-instances-overview

```suggestion
- Enabled an Admin user—this is required for Azure Container Instances.
```